cedar-policy-core/src/entities.rs

```diff
@@ -2621,7 +2621,7 @@ mod schema_based_parsing_tests {
             .from_json_value(entitiesjson)
             .expect_err("should fail due to employee being wrong entity type (missing namespace)");
         assert!(
-            err.to_string().contains(r#"`Employee::"12UA45"` has type `Employee` which is not declared in the schema; did you mean XYZCorp::Employee?"#),
+            err.to_string().contains(r#"`Employee::"12UA45"` has type `Employee` which is not declared in the schema. Did you mean XYZCorp::Employee?"#),
```

Comment:
Should the type in 'Did you mean..' part be surrounded with backticks?

Reply:
Yep, will fix in the next commit.